.gitignore

```diff
@@ -11,3 +11,5 @@ coverage
 # can be used to install node with the correct version when entering the directory
 # https://github.com/direnv/direnv/wiki/Node
 .envrc
+
+/.idea
```

Comment:
Minor: ideally this belongs into your global gitignore file for your user somewhere in your home directory, since it is related to your IDE/local setup